kratos/processes/simple_mortar_mapper_process.cpp

```diff
@@ -143,7 +139,7 @@ void SimpleMortarMapperProcess<TDim, TNumNodes, TVarType, THist>::AssemblyMortar
         {
             PointType global_point;
             SlaveGeometry.GlobalCoordinates(global_point, ConditionsPointSlave[i_geom][i_node]);
-            points_array[i_node] = boost::make_shared<PointType>(global_point);
+            points_array[i_node] = PointType::Pointer( new PointType(global_point) );
```

Comment:
better use make_shared...

Reply:
OK, I was to avoid problem with the future migration of boost, but you are right